gemrb/plugins/SDLVideo/SDLVideo.cpp

```diff
@@ -207,6 +207,10 @@ int SDLVideoDriver::ProcessEvent(const SDL_Event & event)
 				}
 			}
 			break;
+#ifndef USE_SDL_CONTROLLER_API
+		default:
+			break;
+#else
```

Comment:
It will be logical to change the code to
```c++
#ifdef USE_SDL_CONTROLLER_API
...
#else
		default:
			break;
#endif
```
just for consistency

Reply:
the default is superfluous, would you rather i leave it off?

Reply:
No, but you can move it to the end of the switch. It's superfluous only to the compiler.